tapdb/sqlc/queries/assets.sql

```diff
@@ -1016,3 +1017,22 @@ FROM genesis_assets assets
 JOIN assets_meta
     ON assets.meta_data_id = assets_meta.meta_id
 WHERE assets.asset_id = $1;
+
+-- Upsert a record into the mint_anchor_uni_commitments table.
+-- If a record with the same batch_id and group_key already exists, update the
+-- existing record. Otherwise, insert a new record.
+-- name: UpsertMintAnchorUniCommitment :one
+INSERT INTO mint_anchor_uni_commitments (
+    id, batch_id, tx_output_index, taproot_internal_key, group_key
+)
+VALUES ($1, $2, $3, $4, $5)
+ON CONFLICT(batch_id, tx_output_index) DO UPDATE SET
+    tx_output_index = EXCLUDED.group_key,
```

Comment:
`EXCLUDED.group_key,` seems to be wrong! Or you meant to do `group_key = EXCLUDED.group_key`?

Also, since IMO these `ON CONFLICT ... EXCLUDED ...` queries are sometimes a bit hard to read, I got into the habit of adding a small comment above the fields, just mentioning whether it's a NOP (https://github.com/lightninglabs/taproot-assets/blob/74f52ac31c6f9c36ee45e65a85637133c6ac0aed/tapdb/sqlc/queries/assets.sql#L164) or not (https://github.com/lightninglabs/taproot-assets/blob/74f52ac31c6f9c36ee45e65a85637133c6ac0aed/tapdb/sqlc/queries/assets.sql#L175).
I know, in this case it's quite obvious.